src/app/pages/HomePage/ParaTimeSelector/HelpScreen/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import helpScreenNavigate from '../images/help-screen-navigate.svg'
+import helpScreenPinch from '../images/help-screen-pinch.svg'
+import helpScreenTap from '../images/help-screen-tap.svg'
+import { TFunction } from 'i18next'
+import { styled } from '@mui/material/styles'
+import Box from '@mui/material/Box'
+import MobileStepper from '@mui/material/MobileStepper'
+import Typography from '@mui/material/Typography'
+import { COLORS } from '../../../../../styles/theme/colors'
+import { useConstant } from '../../../../hooks/useConstant'
+import SwipeableViews from 'react-swipeable-views'
+import Button from '@mui/material/Button'
+import { ParaTimeSelectorStep } from '../types'
+
+const HelpScreenContainer = styled(Box)(() => ({
+  position: 'absolute',
+  top: '65%',
+  left: '50%',
+  transform: 'translate(-50%, -50%)',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  minHeight: '185px',
+  width: '90%',
+}))
+
+interface Step {
+  icon: string
+  label: string
+}
+
+const steps = (t: TFunction): Step[] => [
+  {
+    icon: helpScreenNavigate,
+    label: t('home.helpScreen.navigate'),
+  },
+  {
+    icon: helpScreenPinch,
+    label: t('home.helpScreen.pinch'),
+  },
+  {
+    icon: helpScreenTap,
+    label: t('home.helpScreen.tap'),
+  },
+]
+
+type AvailableSteps = 0 | 1 | 2
+
+interface HelpScreenProps {
+  setParaTimeStep: (value: ParaTimeSelectorStep) => void
+}
+
+const HelpScreen: FC<HelpScreenProps> = ({ setParaTimeStep }) => {
+  const { t } = useTranslation()
+  const [activeStep, setActiveStep] = useState<AvailableSteps>(0)
+  const allSteps = useConstant(() => steps(t))
+  const totalSteps = allSteps.length
+  const currentStep = allSteps[activeStep]
+
+  const onChangeSwipeableIndex = (index: number) => {
+    setActiveStep(index as AvailableSteps)
+  }
+
+  const onGetStartedClick = () => {
+    setParaTimeStep(ParaTimeSelectorStep.Explore)
+  }
+
+  return (
+    <HelpScreenContainer>
+      <SwipeableViews index={activeStep} onChangeIndex={onChangeSwipeableIndex}>
```

Comment:
Swipe area is very small so I usually end up with moving graph that is below tutorial